experimental/dds/tree2/.dependency-cruiser-known-violations.json

```diff
@@ -490,5 +490,25 @@
 			"src/feature-libraries/editable-tree/editableField.ts",
 			"src/feature-libraries/editable-tree-2/index.ts"
 		]
+	},
+	{
+		"type": "cycle",
+		"from": "src/feature-libraries/editable-tree-2/index.ts",
+		"to": "src/feature-libraries/editable-tree-2/proxies/index.ts",
+		"rule": {
+			"severity": "error",
+			"name": "no-circular"
+		},
+		"cycle": [
+			"src/feature-libraries/editable-tree-2/proxies/index.ts",
+			"src/feature-libraries/editable-tree-2/proxies/node.ts",
+			"src/feature-libraries/editable-tree-2/proxies/proxies.ts",
+			"src/feature-libraries/editable-tree-2/lazyTree.ts",
+			"src/feature-libraries/editable-tree-2/lazyField.ts",
+			"src/feature-libraries/contextuallyTyped.ts",
+			"src/feature-libraries/editable-tree/index.ts",
+			"src/feature-libraries/editable-tree/editableField.ts",
+			"src/feature-libraries/editable-tree-2/index.ts"
+		]
```

Comment:
The cycle is caused by importing 'ContextuallyTypedData' into proxies.ts (for the purposes of conversion).

ContextuallyTypedData transitively depends on proxies.ts via lazy*.